From 4fc65f4fea459f8ab1f99c5ceb575fc8f4ea651a Mon Sep 17 00:00:00 2001
From: Bhushan Shah <bhush94@gmail.com>
Date: Sun, 13 Dec 2020 19:16:30 +0530
Subject: [PATCH] include KDEInstallDirs as first thing

Otherwise ECMGeneratePriFile won't get correct paths and will install
pri file in wrong place and things wanting to use kdb with qmake won't
work.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3219e967..e3f98b43 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,12 +13,12 @@ kdb_add_tests(OFF)
 kdb_add_examples(OFF)
 
 # ECM
+include(KDEInstallDirs)
 include(ECMGeneratePriFile)
 include(ECMInstallIcons)
 include(ECMOptionalAddSubdirectory)
 include(ECMPoQmTools)
 include(ECMSetupVersion)
-include(KDEInstallDirs)
 include(KDECMakeSettings NO_POLICY_SCOPE)
 include(KDEFrameworkCompilerSettings NO_POLICY_SCOPE)
 include(ECMSetupQtPluginMacroNames)
-- 
GitLab

